hardly/handlers/distgit.py

```diff
@@ -89,6 +89,7 @@ def run(self) -> TaskResults:
 Please review the contribution and once you are comfortable with the content,
 you should trigger a CI pipeline run via `Pipelines → Run pipeline`."""
         dg_mr = self.api.sync_release(
+            dist_git_branch=self.package_config.downstream_branch_name,
```

Comment:
Just to be sure: so if `downstream_branch_name` is not specified in the config, nothing changes, i.e. it'll still default to the default branch of the dist-git repository, right?

Reply:
Yes, because sync_release is called with dist_git_branch=None

Reply:
This should be changed, so that when `donstream_branch_name` is not specified use the current source-git branch name as the name of the target branch in dist-git.